lib/OpenCloud/Database/Resource/Instance.php

```diff
@@ -190,13 +201,21 @@ protected function createJson()
             );
         }
 
-        return (object) array(
-            'instance' => (object) array(
+        $out = array(
+            'instance' => array(
                     'flavorRef' => $this->flavor->links[0]->href,
                     'name'      => $this->name,
                     'volume'    => $this->volume
                 )
         );
+
+        if (isset($this->backupRef)) {
+            $out['instance']['restorePoint'] = array(
+                'backupRef' => $this->backupRef
+            );
+        }
+
+        return (object) $out;
     }
 
     /**
```

Comment:
It would be awesome if we had a convenience method that allowed uses to create a backup on a DB instance _itself_, rather than having to construct it. For example:

```php
$instance->createBackup();
```

you could just wrap the above method.